src/components/tools/text-toolbar.tsx

```diff
@@ -5,11 +5,19 @@ import { IFloatingToolbarProps, useFloatingToolbarLocation } from "./hooks/use-f
 import { TextToolbarButton } from "./text-toolbar-button";
 import { IRegisterToolApiProps } from "./tool-tile";
 import { isMac } from "../../utilities/browser";
+import BoldToolIcon from "../../clue/assets/icons/text/bold-text-icon.svg";
+import ItalicToolIcon from "../../clue/assets/icons/text/italic-text-icon.svg";
+import UnderlineToolIcon from "../../clue/assets/icons/text/underline-text-icon.svg";
+import SuperscriptToolIcon from "../../clue/assets/icons/text/superscript-text-icon.svg";
+import SubscriptToolIcon from "../../clue/assets/icons/text/subscript-text-icon.svg";
+import NumberedListToolIcon from "../../clue/assets/icons/text/numbered-list-text-icon.svg";
+import BulletedListToolIcon from "../../clue/assets/icons/text/bulleted-list-text-icon.svg";
```

Comment:
Sorry to do this to you, but I missed it the first time around -- these should be in `src/assets/icons` rather than `clue/assets/icons` because they're not CLUE-specific.